### PR TITLE
[Expert] Alterations to Occultism Ritual outputs

### PR DESCRIPTION
**Pull request by MuteTiefling**
Attempts to improve some of these crafts, to remove the annoyance of crafting.

Logic Director moved away from the Occultism crafts. Makes it available earlier as well. 
image

Remove Sophisticated Backpacks upgrades from Compacting Drawers
image

Ender Storage no longer requires satchels. Similar gating.
image
image
image

Inception Upgrade no longer uses a satchel either:
image

Modular Routers Senders/Pullers
Mk3
image

Mk2s
image
image

Enchantment Library
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
This should resolve #4771, though not in the way requested. 

Effectively all crafts using the satchels have been shifted over to using dimensional storage crystals. This keeps the occultism aspect intact, but serves to reduce the 'annoyance' down to a single craft: the dimensional storage cyrstal itself. The process doesn't change for that, but it does reduce the overall complexity of crafting all of these items.

Similarly, the logic director was removed from occultism entirely and brought forward a touch, as ID's gating has been a constant source of discussion in Discord. 

Finally, Compacting drawers (the other common cause for this all to come up) have had their gating changed, while still maintaining the magic feel of the recipe. Hopefully squashing those issues as well.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/modularrouters/shapeless.js [hunk 1]
```diff
@@ -22,7 +29,7 @@ onEvent('recipes', (event) => {
                 'tconstruct:sky_slime_crystal',
                 'tconstruct:sky_slime_crystal'
             ],
-            id: 'modularrouters:stack_upgrade'
+            id: `'modularrouters:stack_upgrade'`
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/modularrouters/shapeless.js:32]: Typo I think 🤔

**Review by NielsPilgaard — APPROVED** [review]
Excellent changes 😊👍
Feel free to merge when you see fit.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/modularrouters/shapeless.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/modularrouters/shapeless.js]: Well that's a weird place for that to happne >.>

**Pull request merged**